docs/constants/navigation.js

```diff
@@ -1,574 +1,578 @@
+      makePage('modules/module-config.md'),
+    ]),
+  ]),
+  makeSection('Expo Accounts', [
+    makeGroup('Expo Accounts', [
+      makePage('accounts/account-types.md'),
+      makePage('accounts/two-factor.md'),
+      makePage('accounts/programmatic-access.md'),
+      makePage('accounts/working-together.md'),
+    ]),
+  ]),
+  makeSection('Bare Workflow', [
+    makeGroup('Bare Workflow', [
+      makePage('bare/exploring-bare-workflow.md'),
+      makePage('bare/hello-world.md'),
+      makePage('bare/using-libraries.md'),
+      makePage('bare/existing-apps.md'),
+      makePage('bare/installing-expo-modules.md'),
+      makePage('bare/installing-unimodules.md'),
+      makePage('bare/installing-updates.md'),
+      makePage('bare/unimodules-full-list.md'),
+      makePage('bare/using-expo-client.md'),
+      makePage('bare/using-web.md'),
+      makePage('bare/migrating-from-expokit.md'),
+      makePage('bare/updating-your-app.md'),
+      makePage('bare/error-recovery.md'),
+    ]),
+  ]),
+  makeSection('Push Notifications', [
+    makeGroup('Push Notifications', [
+      makePage('push-notifications/overview.md'),
+      makePage('push-notifications/push-notifications-setup.md'),
+      makePage('push-notifications/sending-notifications.md'),
+      makePage('push-notifications/sending-notifications-custom.md'),
+      makePage('push-notifications/receiving-notifications.md'),
+      makePage('push-notifications/using-fcm.md'),
+      makePage('push-notifications/faq.md'),
+    ]),
+  ]),
+  makeSection('Classic Services', [
+    makeGroup('Classic Services', sortAlphabetical(pagesFromDir('classic'))),
+  ]),
+  makeSection('UI Programming', [
+    makeGroup('UI Programming', [
+      makePage('ui-programming/image-background.md'),
+      makePage('ui-programming/implementing-a-checkbox.md'),
+      makePage('ui-programming/z-index.md'),
+      makePage('ui-programming/using-svgs.md'),
+      makePage('ui-programming/react-native-toast.md'),
+      makePage('ui-programming/react-native-styling-buttons.md'),
+    ]),
+  ]),
+  makeSection('Regulatory Compliance', [
+    makeGroup('Regulatory Compliance', sortAlphabetical(pagesFromDir('regulatory-compliance'))),
+  ]),
+  makeSection('Technical Specs', [
+    makeGroup('Technical Specs', [
+      makePage('technical-specs/expo-updates-0.md'),
+      makePage('technical-specs/expo-sfv-0.md'),
+    ]),
+  ]),
+  makeSection('Deprecated', [
+    makeGroup('ExpoKit', [
+      makePage('expokit/overview.md'),
+      makePage('expokit/eject.md'),
+      makePage('expokit/expokit.md'),
+      makePage('expokit/advanced-expokit-topics.md'),
+      makePage('expokit/universal-modules-and-expokit.md'),
+    ]),
+    makeGroup('Archived', sortAlphabetical(pagesFromDir('archived'))),
+  ]),
+  // TODO(cedric): this group isn't visible in the current sidebar, check if we need to fix or remove it
+  makeGroup('Troubleshooting', pagesFromDir('troubleshooting')),
```

Comment:
This is a bit weird, but I can't find this section on our current docs. So I think we should remove it, or fix it and make the pages public again.

Reply:
These pages are now "unaccessible" by the sidebar
- https://docs.expo.dev/troubleshooting/application-has-not-been-registered/
- https://docs.expo.dev/troubleshooting/clear-cache-macos-linux/
- https://docs.expo.dev/troubleshooting/clear-cache-windows/
- https://docs.expo.dev/troubleshooting/react-native-version-mismatch/

Reply:
i dont see these referenced in expo-cli or anywhere else in universe, so I think it would be safe to remove them. (they are forever preserved in git if we need to dig them up)